### PR TITLE
Document the Decide class

### LINKED ISSUE #85: doc; Add documentation to the Decide class
Documentation is missing for the decide class

### PR DESCRIPTION
**Pull request by Frequinzy**
Fix #85

### CODE REVIEW

**Review by LottaJohnsson — APPROVED** [review]
LGTM

File: src/decide/Decide.java [hunk 1]
```diff
@@ -5,7 +5,22 @@
...
 
+/**
+ * Makes the final decision on wheter to allow a launch or not. If the input parameters 
+ * are invalid this will throw an exception. Returns it's answer in the form of
+ * "YES" or "NO" in standard output.
+ */
 public class Decide {
+    /**
+     * Decides wheter to allow a launch or not. 
+     * NOTE: Retruns it's answer in the form of "YES" or "NO" in standard output.
```
Comment (annamaartensson) [inline src/decide/Decide.java:16]: Fix some typos: *Returns, *its, *whether

**Review by annamaartensson — COMMENTED** [review]
LGTM besides typos

**Review by annamaartensson — APPROVED** [review]
LGTM

**Pull request merged**